OneDrive-Cloud-Player/Services/MediaTrackService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using LibVLCSharp.Shared;
+using LibVLCSharp.Shared.Structures;
+using OneDrive_Cloud_Player.Exceptions;
+using OneDrive_Cloud_Player.Models.Interfaces;
+using System.Diagnostics;
+using System.Linq;
+using Windows.Storage;
+
+namespace OneDrive_Cloud_Player.Services
+{
+    public class MediaTrackService : IMediaTrackService
+    {
+        private MediaPlayer _mediaPlayer;
+        private bool _isInitialized = false;
+        private readonly ApplicationDataContainer _userSettings;
+
+        /// <summary>
+        /// Manage the media tracks of the libvlcsharp media player instance.
```

Comment:
should this be the constructor doc or class-wide doc?

Reply:
It can be moved to class